app-editors/komodo-edit-bin/komodo-edit-bin-10.2.2-r1.ebuild

```diff
@@ -44,29 +43,33 @@ RDEPEND="${DEPEND}
 	gnome-base/libgnomeui
 	gnome-base/libgnome"
 
-RESTRICT="mirror"
-
```

Comment:
I think that was a double entry that would have negated RESTRICT="mirror strip" I had above. The source *is not* redistributable.